<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>842</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>2D Grafik İşlemleri</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icon-matplotlib.ico</normaloff>../images/icon-matplotlib.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPlainTextEdit{
	border:solid 4px black;
	background:#ffffff;
}

QPushButton,
QToolButton{
	background-color:gray;
	color:white;
	font-size:13px;
	border: 2px solid gray;
	border-radius: 5px;
}
 

QPushButton:hover,
QToolButton:hover{
	background:white; 
	color:gray; 
}

QPushButton#btnOpenFile{
	background-color:#f40352;
	border: 2px solid #f40352;
}

QPushButton#btnOpenFile:hover{
	background:white;  
	color:#f40352;
}

.clsDeneme{
color:red
}
 </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>991</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&quot;matplotlib&quot; Kütüphanesi ile 2D Grafik/Resim İşlemleri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Resim Dosyası Seç</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>611</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Dosya Adresi</string>
   </property>
   <widget class="QLabel" name="lblPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Resim Boyutu</string>
   </property>
   <widget class="QLabel" name="lblProperties">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>_</string>
    </property>
    <property name="class" stdset="0">
     <string>clsDeneme clsDeneme234</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>1061</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sana ne&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Image Plotting</string>
    </attribute>
    <widget class="QPlainTextEdit" name="txtEdit1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>1021</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="documentTitle">
      <string/>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="class" stdset="0">
      <string>clsTxtEditor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Komutu Çalıştır</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chcGrid_Plot">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid çizgilerini göster</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Applying Pseudocolor</string>
    </attribute>
    <widget class="QPlainTextEdit" name="txtEdit2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>1011</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Komutu Çalıştır</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ddlCmapOptions">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Uygulanacak Renk Paleti</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chcGrid_Pseudocolor">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid çizgilerini göster</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Histogram</string>
    </attribute>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Komutu Çalıştır</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gray Scale</string>
    </attribute>
    <widget class="QPushButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Komutu Çalıştır</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chcGrid_GrayScale">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid çizgilerini göster</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Split RGB Channels</string>
    </attribute>
    <widget class="QPushButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Komutu Çalıştır</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QTextEdit" name="txtHtmlContentHolder">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>761</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>3D Data Processing</string>
    </attribute>
    <widget class="QPushButton" name="btnOpen3DdataFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>3D Veri Dosyası Seç</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>671</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Data Boyutu</string>
     </property>
     <widget class="QLabel" name="lbl3D_W">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>_</string>
      </property>
      <property name="class" stdset="0">
       <string>clsDeneme clsDeneme234</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Images : </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl3D_H">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>_</string>
      </property>
      <property name="class" stdset="0">
       <string>clsDeneme clsDeneme234</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl3D_Z">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>_</string>
      </property>
      <property name="class" stdset="0">
       <string>clsDeneme clsDeneme234</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Voxel Height : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Number_of_Slices">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Slices : </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl3D_voxel_Width">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>_</string>
      </property>
      <property name="class" stdset="0">
       <string>clsDeneme clsDeneme234</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl3D_Number_of_Slices">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>_</string>
      </property>
      <property name="class" stdset="0">
       <string>clsDeneme clsDeneme234</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Voxel Width : </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl3D_voxel_Height">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>_</string>
      </property>
      <property name="class" stdset="0">
       <string>clsDeneme clsDeneme234</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
